### PR TITLE
Help: Pattern Guide: Better explanation of loop sample tempo handling

### PR DESCRIPTION
**Pull request by jamshark70**
## Purpose and Motivation

A forum question revealed that the discussion of sample looping in the Pattern Guide is unclear. It uses a "magic number" for tempo, without explaining how to derive it.

The discussion was also incomplete -- no information about how to adjust to a different playback tempo.

Remedied by splitting the procedure up into three steps.

## Types of changes

- Documentation

## To-do list

- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp [hunk 1]
```diff
@@ -35,21 +76,22 @@ SynthDef(\bell, { |out, accent = 0, amp = 0.1, decayScale = 1|
...
 		], exc, freqscale: accent + 1, decayscale: decayScale);
 	DetectSilence.ar(sig, doneAction: Done.freeSelf);
-	Out.ar(out, sig ! 2)
+	Out.ar(out, sig.dup)
 }).add;
 )
 
 (
-TempoClock.default.tempo = 0.35953685899971 * 4;
+TempoClock.tempo = 1.4381474359988;
```
Comment (snappizz) [inline HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp]: how about setting `start` and `end` variables and writing `TempoClock.tempo = ((end - start) / numBeats).reciprocal`? that seems like a better recommended practice than a hardcoded number.

Comment (jamshark70) [reply HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp]: OK, with a tighter math expression.

File: HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp [hunk 2]
```diff
@@ -20,10 +61,13 @@ code::
 b = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
 
 // one loop segment
-SynthDef(\oneLoop, { |out, bufnum, start, time, amp|
-	var	sig = PlayBuf.ar(1, bufnum, startPos: start, loop: 0),
-		env = EnvGen.kr(Env.linen(0.01, time, 0.05, level: amp), doneAction: Done.freeSelf);
-	Out.ar(out, (sig * env) ! 2)
+SynthDef(\oneLoop, { |out, bufnum, start, time, amp, rate = 1|
+	var	sig = PlayBuf.ar(1, bufnum,
```
Comment (patrickdupuis) [inline HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp]: It looks like there's a tab between `var` and `sig = ...`. This makes for some strange indentation in the HelpBrowser. Would you mind replacing them with a space here and elsewhere?

Comment (jamshark70) [reply HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp]: OK, fixed.

**Pull request merged**